### PR TITLE
Remove accept ownership checkbox from Details.tsx

### LINKED ISSUE #165: Remove Accept Ownership checkbox in upload page
It's an extra step, we can remove that checkbox.

Image

### PR DESCRIPTION
**Pull request by ujwalkumar1995**
## Description

Closes #165 

## Type of change

- [x] Minor changes in old functionality

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas -- (N/A as only have deleted old code not added any new code)
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by sasicodes** [comment]
Please fill out the description acknowledgments 🙇🏻

File: src/components/Upload/Details.tsx [hunk 1]
```diff
@@ -183,25 +183,6 @@ const Details: FC<Props> = ({ onUpload, onCancel }) => {
               />
             </div>
           </div>
```
Comment (sasicodes) [inline src/components/Upload/Details.tsx:185]: Make sure `acceptTerms` property in the form initialization and default values were removed.

Comment (ujwalkumar1995) [reply src/components/Upload/Details.tsx:185]: Ok will look into this and try to fix this.

Comment (ujwalkumar1995) [reply src/components/Upload/Details.tsx:181]: Have made the change as suggested by you. Is it fine now or I am missing something?

**Review by sasicodes — APPROVED** [review]
Yugi Thumb

**Pull request merged**